### ISSUE TITLE
Naturbruk

### ISSUE THREAD

**Issue by danmichaelo** [realfagstermer/realfagstermer#1599]
Trukket ut fra #1580:

Denne er brukt på tre dokumenter; ett om hvordan barn bruker naturen, ett om naturforvaltning og ett om plantevern. Trenger vi termen? Hvordan vil vi i så fall definere den?

https://app.uio.no/ub/emnesok/realfagstermer/search?uri=http:%2F%2Fdata.ub.uio.no%2Frealfagstermer%2Fc007769

**Comment by Hypsibius** [comment]
Godt spørsmål, men det er jo eit studium på VGS

https://www.vilbli.no/nb/nb/no/naturbruk/program/v.na

**Comment by Hypsibius** [comment]
Det blir jo ein parallell til landbruk, tenkjer eg, men omfattar tydelegvis reindrift og fiske

**Comment by BioHeidi** [comment]
Ja, tolker den også som noe videre enn landbruk, og kunne sikkert vært brukt på flere dokumenter.
